app/workers/mail_chimp_worker.rb

```diff
@@ -0,0 +1,21 @@
+class MailChimpWorker
+  include Sidekiq::Worker
+
+  def perform(user_id)
+    user= User.find(user_id)
+    subscribe_to_mailchimp(ENV['USERS_LIST_ID'], user)
+    if user_id.subscribe_newsletter
```

Comment:
this would throw an error since `user_id` is an integer and i think you meant `user`

Reply:
but, we wont be needing this check here since we would be doing it in the model before calling this worker,